Improve FT.SPELLCHECK doc on TERMS option (#3166)
RediSearch/RediSearch: docs/commands/ft.spellcheck.md

<details open>
<summary><code>TERMS</code></summary> 

specifies an inclusion (`INCLUDE`) or exclusion (`EXCLUDE`) of a custom dictionary. Refer to `FT.DICTADD`, `FT.DICTDEL` and `FT.DICTDUMP` about managing custom dictionaries.

> Multiple inclusion and/or exclusion `TERMS` can be specified.

Reviewer: ```suggestion
You can specify multiple inclusion and exclusion `TERMS`.
```
